schemas/top-api.yml

```diff
@@ -438,6 +438,155 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - conceptcluster
+  /conceptgraphs/pipeline:
+    parameters:
+      - $ref: '#/components/parameters/include'
+    post:
+      operationId: startConceptGraphPipeline
+      summary: 'Initializes a complete pipeline with all steps ''preprocessing'', ''embedding'', ''clustering'' & ''graph creation'''
+      parameters:
+        - name: process
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: lang
+          in: query
+          schema:
+            type: string
+        - name: skipPresent
+          in: query
+          schema:
+            type: boolean
+            default: true
+        - name: returnStatistics
+          in: query
+          schema:
+            type: boolean
+            default: false
+      requestBody:
+        description: Upload configs and data file.
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: string
+                  format: binary
+                labels:
+                  type: string
+                  format: binary
+                dataConfig:
+                  type: string
+                  format: binary
+                embeddingConfig:
+                  type: string
+                  format: binary
+                clusteringConfig:
+                  type: string
+                  format: binary
+                graphConfig:
+                  type: string
+                  format: binary
+              required:
+                - data
+      responses:
+        '200':
```

Comment:
Why there are both 200 and 202 responses?

Reply:
You can start the pipeline process in such a way that it either waits for it being finished (you get a 200 response with graph statistics) or it just starts in the background and returns a 202 (with pipeline status response)

Reply:
Is this controlled with query parameter "returnStatistics"?

Reply:
Documentation has been updated accordingly.